### PR TITLE
[Vote] Governance update around maintainership 

### PR DESCRIPTION
**Pull request by alvinlin123**
I would like to call an official vote to update the governance with changes around maintainership. Please see  the diff in this PR for the changes. 

Per [existing governance rule](https://cortexmetrics.io/docs/contributing/governance/#changes-in-governance), 2/3 majority votes are required for the changes to pass. 

Please cast your yes/no vote as comment in this PR, I will leave the vote open for 14 days (including weekends) or until all maintainer voted; whichever comes first.

### CODE REVIEW

**Comment by alvinlin123** [comment]
+1 binding.

**Review by alanprot — APPROVED** [review]

**Comment by alanprot** [comment]
+1 binding.

File: GOVERNANCE.md [hunk 1]
```diff
@@ -7,13 +7,30 @@ This document defines project governance for the project.
 The Cortex project employs voting to ensure no single member can dominate the project. Any maintainer may cast a vote. To avoid having a single company dominate the project, at most two votes from maintainers working for the same company will count.
 
 For formal votes, a specific statement of what is being voted on should be added to the relevant github issue or PR, and a link to that issue or PR added to the maintainers meeting agenda document.
-Maintainers should indicate their yes/no vote on that issue or PR, and after a suitable period of time, the votes will be tallied and the outcome noted.
+
+Maintainers should indicate their yes/no vote on that issue or PR, and after a suitable period of time (minimum 2 business weeks), the votes will be tallied and the outcome noted. Mainainers who do not cast a vote, after a suitable period of time, are not included in the marjority calculation.
```
Comment (friedrichg) [inline GOVERNANCE.md:11]:
```suggestion
Maintainers should indicate their yes/no vote on that issue or PR, and after a suitable period of time (minimum 2 business weeks), the votes will be tallied and the outcome noted. Maintainers who do not cast a vote, after a suitable period of time, are not included in the majority calculation.
```

**Review by friedrichg — CHANGES_REQUESTED** [review]
Just some spelling corrections, otherwise good :+1:

**Review by friedrichg — APPROVED** [review]

**Review by jtlisi — APPROVED** [review]
+1 binding

**Comment by alvinlin123** [comment]
With 4/4 +1s, I declare the governance update is effective as of now :)

**Pull request merged**